xmake/modules/lib/detect/find_package.lua

```diff
@@ -83,9 +85,15 @@ function main(name, opt)
     -- trace
     if opt.verbose or option.get("verbose") then
         if result then
-            cprint("checking for the %s ... ${color.success}%s", name, result.version and result.version or "${text.success}")
+            -- only display manager of found package if the package we searched for
+            -- did not specify a package manager
+            local original_manager_name, _, _ = split_package_name(name)
+            local display_name = original_manager_name and package_name
+                or (manager_name .. "::" .. package_name)
+
+            cprint("checking for %s ... ${color.success}found %s %s", name, display_name, result.version and result.version or "")
```

Comment:
I think there is no need to add `found` word.

Reply:
No problem, removed.